docs/developers/docker_gluten_centos7.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+title: Docker script for CentOS 7
+nav_order: 7
+parent: Developers
+grand_parent: Documentations
+permalink: /docs/developers/docker-centos7/
+---
+Here is a docker script we verified to build Gluten+Velox backend on Pre-build CentOS 7:
+
+Run on host as root user:
+```bash
+docker pull apache/gluten:vcpkg-centos-7
+docker run -itd --name gluten-centos-7 apache/gluten:vcpkg-centos-7 /bin/bash
+docker attach gluten
+```
+
+Run in docker:
+```bash
+git clone https://github.com/apache/incubator-gluten.git gluten
+cd gluten
+
+# To access HDFS or S3, you need to add the parameters `--enable_hdfs=ON` and `--enable_s3=ON`
+# If you have the same error with issue-3283, you need to add the parameter `--compile_arrow_java=ON`
+./dev/buildbundle-veloxbe.sh
```

Comment:
`./dev/ci-velox-buildstatic-centos-7.sh` not work:
```
...
Could not find artifact org.apache.arrow:arrow-memory-unsafe:jar:15.0.0-gluten in central
...
```

`./dev/build_arrow.sh` also not work:
```
CMake Error at CMakeLists.txt:95 (project):
  No CMAKE_CXX_COMPILER could be found.

  Tell CMake where to find the compiler by setting either the environment
  variable "CXX" or the CMake cache entry CMAKE_CXX_COMPILER to the full path
  to the compiler, or to the compiler name if it is in the PATH.
```

Reply:
It's maybe my local env issue. `./dev/ci-velox-buildstatic-centos-7.sh` works if running on github action.